internal/cmd/schema-registry/command_region.go

```diff
@@ -6,12 +6,20 @@ import (
 	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
 )
 
-type schemaRegistryCloudRegion struct {
-	ID          string   `human:"ID" serialized:"id"`
-	Cloud       string   `human:"Cloud" serialized:"cloud"`
-	RegionName  string   `human:"Region Name" serialized:"region_name"`
-	DisplayName string   `human:"Display Name" serialized:"display_name"`
-	Packages    []string `human:"Packages" serialized:"packages"`
+type schemaRegistryCloudRegionHumanOut struct {
+	ID         string `human:"ID"`
+	Name       string `human:"Name"`
+	Cloud      string `human:"Cloud"`
+	RegionName string `human:"Region Name"`
```

Comment:
```suggestion
	Region string `human:"Region"`
```

Reply:
Don’t we need to keep consistency between the public REST API and the CLI in terms of the field names? In the rest api we call them - ‘Display Name’ and ‘Region Name’.

Ref - https://docs.confluent.io/cloud/current/api.html#tag/Regions-(srcmv2)/operation/getSrcmV2Region

Reply:
It seems a bit odd to write "Display Name" while we're displaying it, haha. I could be convinced to use "Region Name", since it doesn't matter too much, but in general we prefer concise names over verbose names.

Reply:
Agreed. I have update 'Display Name' to 'Display', we can probably keep the 'Region Name' as is!